src/index.html

```diff
@@ -11,7 +11,176 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <fieldset class="form-field block">
+        <legend>Personal information</legend>
+        <div class="input">
+          <label for="aaa">Surname:</label>
+          <input 
+          type="text" 
+          name="attribute" 
+          autocomplete="off" 
+          required
+          >
```

Comment:
```suggestion
          <input 
            type="text" 
            name="attribute" 
            autocomplete="off" 
            required
          >
```
Fix all cases
![image](https://user-images.githubusercontent.com/96409563/222083146-d3a3f010-32f3-45c3-aa7c-02219581f3c3.png)